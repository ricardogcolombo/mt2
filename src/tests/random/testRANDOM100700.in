100 700
1 92 55 88 69
1 8 48 100 87
1 61 123 3 51
1 71 41 86 69
1 86 13 41 91
1 25 55 28 113
1 12 102 25 38
1 82 14 38 32
1 68 93 47 91
1 39 38 32 122
1 46 90 75 35
1 38 62 27 64
1 88 25 94 5
1 86 54 4 101
1 65 113 46 47
1 60 17 70 115
1 30 36 26 0
1 20 18 95 126
1 80 45 16 107
1 90 117 4 98
1 35 79 19 77
1 14 122 25 21
1 31 59 46 101
1 13 44 51 18
1 21 19 65 13
1 69 54 92 90
1 71 40 7 27
1 93 29 40 43
1 6 16 33 88
1 100 127 91 128
1 88 69 6 50
1 5 110 9 7
1 35 4 32 42
1 65 57 34 29
1 58 109 17 22
1 67 93 42 46
1 37 107 10 93
1 72 39 20 81
1 56 7 75 50
1 4 106 98 111
1 36 99 50 68
1 58 42 33 85
1 59 47 50 23
1 42 118 94 14
1 60 59 91 60
1 72 48 68 85
1 79 121 15 33
1 43 95 92 52
1 3 103 24 36
1 41 79 45 101
1 26 59 39 44
1 78 59 6 2
1 93 67 31 73
1 97 39 23 12
1 71 7 94 22
1 28 66 23 15
1 52 41 37 65
1 30 86 78 109
1 87 64 22 90
1 49 52 75 54
1 12 58 46 103
1 2 67 98 38
1 94 104 27 49
1 92 72 15 79
1 53 77 77 7
1 50 21 71 48
1 59 41 32 22
1 87 11 17 31
1 6 26 79 111
1 86 83 39 29
1 92 92 22 29
1 44 55 100 92
1 72 19 52 34
1 65 60 92 116
1 41 55 88 107
1 22 124 23 2
1 37 16 19 68
1 11 123 73 15
1 16 15 86 97
1 80 90 83 50
1 31 120 16 75
1 76 27 82 100
1 85 76 59 19
1 12 118 4 62
1 46 63 89 15
1 72 126 69 128
1 30 79 50 115
1 5 34 36 109
1 31 86 58 125
1 82 45 3 36
1 97 67 51 119
1 53 75 24 93
1 41 111 7 2
1 27 101 10 118
1 83 33 71 60
1 79 70 41 47
1 62 106 43 4
1 11 43 37 54
1 76 123 22 124
1 20 79 62 0
1 97 100 96 33
1 99 117 87 46
1 44 17 99 24
1 15 109 88 38
1 20 53 100 32
1 77 107 17 7
1 100 110 40 41
1 65 36 100 29
1 20 14 10 81
1 90 79 58 30
1 27 69 46 56
1 72 20 63 39
1 25 124 43 49
1 84 21 72 26
1 75 3 87 105
1 77 46 27 92
1 21 71 56 62
1 41 36 30 5
1 30 98 76 81
1 17 110 27 88
1 77 45 55 2
1 2 95 58 128
1 51 114 23 75
1 86 69 38 123
1 85 46 7 124
1 59 70 40 39
1 75 70 72 0
1 14 48 7 107
1 8 124 90 121
1 16 81 33 119
1 34 14 96 106
1 100 84 80 120
1 97 32 17 5
1 9 110 5 26
1 63 112 50 107
1 20 62 85 78
1 27 90 18 43
1 26 18 48 97
1 86 129 63 14
1 38 97 44 128
1 6 38 97 118
1 70 33 11 55
1 74 121 11 122
1 20 117 41 25
1 35 107 63 51
1 30 118 57 130
1 57 4 78 65
1 5 96 31 50
1 90 23 92 46
1 77 81 21 75
1 11 2 90 73
1 80 64 41 65
1 49 56 2 105
1 21 65 78 67
1 45 22 58 111
1 10 65 35 37
1 5 18 26 86
1 55 45 93 88
1 47 20 27 68
1 60 45 9 75
1 36 103 62 65
1 93 114 3 24
1 60 114 85 3
1 34 77 64 126
1 92 88 93 83
1 71 88 2 71
1 62 1 41 16
1 83 76 66 74
1 13 31 28 9
1 22 72 13 44
1 53 49 27 15
1 82 85 46 8
1 48 102 69 36
1 19 86 69 110
1 68 16 16 9
1 56 32 85 15
1 22 1 99 46
1 61 56 22 77
1 43 30 98 10
1 39 129 95 79
1 19 7 90 122
1 65 44 13 5
1 1 22 75 127
1 2 72 87 96
1 82 26 11 69
1 17 53 18 109
1 48 27 32 94
1 90 73 20 90
1 22 111 35 83
1 78 69 27 115
1 54 19 18 103
1 93 23 26 55
1 46 88 14 27
1 80 59 60 113
1 88 105 42 26
1 35 54 48 99
1 14 43 7 90
1 38 26 24 30
1 63 89 49 22
1 8 129 42 33
1 52 123 30 19
1 8 47 52 86
1 80 119 43 33
1 47 12 65 85
1 3 1 35 84
1 34 48 42 106
1 49 92 56 69
1 2 93 50 23
1 92 91 56 46
1 14 50 75 87
1 14 36 24 117
1 88 92 44 15
1 14 0 81 9
1 76 57 92 28
1 2 123 7 41
1 49 22 98 11
1 30 45 25 64
1 81 34 50 112
1 90 54 71 97
1 42 61 83 130
1 7 81 91 25
1 29 77 1 93
1 99 81 20 34
1 47 94 57 93
1 93 42 21 2
1 98 23 90 45
1 11 79 46 60
1 36 99 70 32
1 62 41 23 31
1 62 119 12 114
1 38 100 81 103
1 27 71 100 92
1 78 21 93 121
1 18 80 9 84
1 100 73 83 70
1 79 31 58 57
1 51 68 72 16
1 70 13 82 5
1 5 121 6 129
1 79 114 47 78
1 22 95 34 102
1 11 77 65 31
1 45 26 35 11
1 71 19 59 90
1 37 34 16 37
1 38 36 50 86
1 64 31 27 36
1 95 87 47 1
1 12 79 82 6
1 43 85 69 129
1 93 34 16 69
1 74 105 96 129
1 4 118 36 46
1 30 102 21 50
1 66 13 51 75
1 47 99 90 41
1 77 7 63 46
1 38 123 43 17
1 11 10 10 43
1 34 71 6 113
1 50 114 49 28
1 94 17 86 122
1 4 109 94 101
1 17 55 12 88
1 25 98 50 128
1 75 126 73 90
1 69 99 14 74
1 26 85 84 58
1 11 75 100 71
1 19 121 80 123
1 95 112 25 102
1 50 55 83 69
1 31 114 10 97
1 78 113 62 69
1 90 16 77 37
1 92 79 58 38
1 57 112 36 74
1 13 57 25 98
1 88 21 29 68
1 52 123 65 92
1 93 35 94 19
1 46 126 62 97
1 15 102 60 10
1 41 95 28 106
1 67 66 100 128
1 77 88 7 38
1 46 70 67 23
1 45 5 68 49
1 36 85 96 103
1 24 101 77 67
1 29 121 3 92
1 41 73 6 2
1 74 100 87 108
1 12 28 87 32
1 46 67 63 123
1 69 56 86 119
1 26 116 91 10
1 74 51 99 107
1 87 41 10 7
1 73 81 11 99
1 92 5 20 16
1 41 60 2 13
1 88 11 58 23
1 37 13 66 60
1 40 105 48 104
1 16 47 95 55
1 14 96 33 38
1 44 117 76 122
1 45 118 7 34
1 96 97 47 38
1 33 32 67 93
1 53 103 18 83
1 40 14 71 65
1 64 45 35 16
1 19 59 24 118
1 49 39 81 78
1 49 69 45 93
1 79 58 61 121
1 42 16 15 81
1 50 53 100 17
1 75 27 48 82
1 38 102 55 82
1 36 71 57 7
1 73 123 84 117
1 32 104 11 94
1 49 52 50 91
1 70 120 58 130
1 18 117 50 124
1 77 40 81 122
1 42 51 75 0
1 60 116 14 109
1 37 32 62 87
1 83 125 89 69
1 78 90 13 7
1 38 73 21 1
1 16 126 93 21
1 92 104 52 85
1 14 5 75 51
1 85 94 87 121
1 99 126 23 130
1 74 70 21 53
1 19 12 87 86
1 74 93 14 47
1 30 109 53 66
1 63 55 15 1
1 57 27 49 26
1 72 33 4 11
1 70 130 93 93
1 97 68 92 60
1 34 17 59 38
1 40 39 87 88
1 39 86 3 112
1 91 26 51 101
1 78 19 96 126
1 89 97 60 32
1 61 52 13 107
1 74 128 90 124
1 38 0 91 39
1 56 129 97 3
1 54 13 27 61
1 22 23 6 129
1 56 84 18 53
1 66 117 71 107
1 90 32 92 119
1 56 94 29 30
1 86 72 55 105
1 96 14 68 89
1 67 12 97 122
1 100 27 9 98
1 47 112 50 23
1 71 1 59 82
1 13 130 50 119
1 87 116 91 46
1 31 29 71 11
1 36 103 88 38
1 82 104 34 63
1 76 39 86 105
1 10 94 45 87
1 1 37 63 83
1 71 3 93 119
1 64 25 6 45
1 60 97 45 68
1 34 113 9 10
1 22 17 68 4
1 44 70 68 128
1 97 4 37 126
1 37 98 55 76
1 36 103 19 111
1 1 33 35 70
1 4 33 10 47
1 75 24 90 85
1 31 95 79 64
1 37 90 94 35
1 87 46 48 2
1 80 12 64 74
1 98 77 72 118
1 8 41 3 115
1 59 128 55 82
1 44 67 76 72
1 78 89 58 42
1 30 51 77 86
1 83 89 39 75
1 30 96 45 91
1 66 41 72 18
1 100 130 27 34
1 84 56 25 77
1 74 21 1 96
1 23 105 99 85
1 23 118 33 120
1 46 89 38 46
1 20 72 97 120
1 90 125 53 86
1 82 95 23 50
1 98 20 21 89
1 91 51 84 79
1 60 34 90 54
1 73 55 54 0
1 56 81 24 42
1 50 68 79 33
1 1 94 37 120
1 12 10 38 25
1 28 114 25 129
1 54 63 61 11
1 14 52 22 67
1 95 112 86 41
1 59 7 86 74
1 76 130 31 66
1 61 28 97 13
1 78 97 73 22
1 3 69 57 25
1 73 34 72 31
1 22 9 81 47
1 43 0 15 8
1 64 54 36 114
1 80 103 41 82
1 84 39 66 71
1 7 103 75 36
1 85 66 100 14
1 100 105 29 104
1 73 76 48 31
1 22 7 73 120
1 38 32 85 90
1 47 111 50 45
1 99 104 55 37
1 64 128 35 18
1 61 53 42 63
1 63 108 72 104
1 56 26 34 48
1 78 121 85 88
1 87 18 9 47
1 66 71 70 110
1 58 102 72 93
1 60 34 87 50
1 47 80 65 54
1 69 98 73 68
1 75 59 73 113
1 29 120 92 4
1 47 1 91 122
1 41 123 21 21
1 12 46 5 124
1 15 111 31 104
1 30 99 7 126
1 36 1 41 50
1 16 103 66 83
1 23 90 95 109
1 30 63 39 13
1 14 0 40 33
1 25 8 20 110
1 10 3 38 52
1 88 62 70 126
1 1 78 58 84
1 3 111 32 87
1 58 26 84 49
1 46 66 56 113
1 80 100 34 109
1 47 36 68 12
1 3 112 93 2
1 40 130 84 56
1 51 39 11 45
1 34 115 61 75
1 91 125 77 48
1 26 98 50 70
1 60 0 31 20
1 52 100 43 40
1 71 116 59 113
1 76 69 64 1
1 61 39 87 100
1 92 40 99 27
1 53 80 52 101
1 50 41 44 94
1 17 17 57 7
1 26 49 12 68
1 99 78 52 60
1 48 50 60 124
1 75 89 33 10
1 61 62 84 25
1 69 28 94 33
1 65 74 1 13
1 43 37 78 114
1 27 108 20 112
1 4 128 30 13
1 67 130 49 26
1 48 97 10 84
1 75 6 13 118
1 3 56 79 2
1 18 100 56 62
1 50 24 66 76
1 31 38 33 12
1 10 75 91 32
1 88 114 4 33
1 13 77 14 68
1 98 69 75 56
1 98 19 65 98
1 38 84 91 30
1 13 1 4 125
1 87 62 70 100
1 46 27 56 77
1 58 124 35 93
1 60 10 25 115
1 84 32 98 66
1 1 60 45 27
1 32 69 7 88
1 70 78 20 58
1 26 106 77 51
1 69 7 59 72
1 39 106 65 124
1 19 83 28 98
1 67 58 9 71
1 65 46 47 126
1 98 108 22 123
1 60 79 32 34
1 31 46 80 49
1 87 96 44 13
1 36 120 75 85
1 32 114 30 36
1 18 115 99 44
1 41 46 51 15
1 30 96 49 67
1 46 122 79 127
1 75 1 65 55
1 85 9 9 102
1 73 58 60 84
1 84 37 5 58
1 100 107 95 47
1 90 10 34 49
1 70 80 79 1
1 82 37 92 29
1 9 11 35 48
1 5 60 92 50
1 10 104 32 43
1 64 113 99 123
1 43 99 30 9
1 32 86 1 68
1 53 68 7 91
1 31 18 62 59
1 5 47 67 94
1 50 43 70 130
1 9 76 28 25
1 1 83 53 115
1 29 77 5 92
1 41 38 60 124
1 31 2 70 67
1 12 130 32 115
1 18 42 64 7
1 38 107 41 117
1 23 54 93 101
1 98 27 16 23
1 92 34 70 79
1 10 68 51 22
1 96 57 9 70
1 84 98 42 109
1 59 6 13 67
1 7 4 74 123
1 67 76 49 66
1 59 104 63 46
1 82 67 57 2
1 64 0 12 75
1 75 59 11 7
1 38 97 11 77
1 18 129 1 45
1 3 12 17 47
1 36 83 47 50
1 24 111 22 6
1 97 95 70 92
1 45 23 82 30
1 49 51 6 13
1 58 17 27 114
1 4 3 51 117
1 43 44 56 24
1 47 127 95 100
1 6 16 18 75
1 40 125 17 111
1 74 88 93 98
1 1 96 42 107
1 61 52 53 8
1 62 61 13 49
1 32 88 6 52
1 92 9 7 12
1 36 114 31 6
1 28 65 7 10
1 45 101 41 60
1 10 76 97 41
1 48 15 74 104
1 96 80 52 88
1 50 21 25 124
1 42 110 81 113
1 47 65 23 82
1 11 85 43 94
1 92 87 57 121
1 39 55 19 32
1 9 28 72 114
1 89 42 80 14
1 76 11 75 45
1 79 48 3 47
1 35 14 65 60
1 79 119 63 114
1 44 37 46 28
1 56 93 50 76
1 5 104 56 13
1 25 89 60 51
1 74 98 15 53
1 4 10 72 70
1 17 25 74 15
1 41 101 13 72
1 59 11 81 25
1 48 5 7 60
1 76 101 67 46
1 87 67 57 76
1 99 104 64 129
1 15 36 22 3
1 21 13 60 53
1 90 60 19 0
1 58 28 1 83
1 9 31 14 95
1 61 54 87 62
1 19 73 60 117
1 55 125 15 18
1 27 84 87 109
1 59 98 18 13
1 87 12 81 91
1 75 56 70 62
1 16 24 41 118
1 13 48 34 31
1 94 76 79 0
1 22 113 60 36
1 86 115 27 42
1 10 15 27 62
1 86 96 28 117
1 63 36 91 65
1 99 114 22 70
1 96 105 36 54
1 3 50 89 101
1 95 61 89 8
1 9 1 59 68
1 63 104 97 8
1 7 117 74 22
1 38 124 75 101
1 71 97 86 113
1 78 74 49 20
1 51 119 85 122
1 31 17 4 95
1 33 109 88 111
1 13 88 7 62
1 54 118 10 12
1 49 36 55 62
1 67 95 11 25
1 90 106 77 55
1 82 37 89 115
1 93 114 14 24
1 18 85 46 108
1 80 43 94 80
1 38 61 100 119
1 70 101 9 13
1 97 84 88 8
1 57 106 26 54
1 26 74 28 42
1 72 77 57 120
1 32 55 42 120
1 20 97 72 102
1 46 25 28 87
1 92 87 31 88
1 49 4 36 69
1 32 17 87 110
1 10 43 43 126
1 92 127 31 56
1 70 19 81 85
1 22 130 79 54
1 62 8 67 44
1 48 22 8 125
1 69 92 66 98
1 85 100 51 49
1 95 94 74 66
1 35 117 7 0
1 30 22 51 30
1 41 9 32 109
1 35 129 93 31
1 74 123 58 32
1 96 94 53 18
1 65 97 66 43
1 96 124 2 98
