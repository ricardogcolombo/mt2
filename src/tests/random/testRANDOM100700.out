0.299934
0.441558
0.460642
0.424685
0.568137
0.335604
0.494167
0.513034
0.593384
0.555356
0.549764
0.598980
0.567374
0.221412
0.608216
0.443214
0.489393
0.477750
0.225290
0.571689
0.295045
0.552084
0.341486
0.627331
0.540415
0.598105
0.549335
0.621337
0.437906
0.355997
0.442527
0.483398
0.393090
0.465326
0.417504
0.635148
0.599091
0.519261
0.283832
0.522369
0.570887
0.528095
0.447969
0.382020
0.410913
0.483901
0.530390
0.397794
0.201445
0.404896
0.531049
0.671748
0.386186
0.424986
0.387539
0.469253
0.551896
0.510871
0.691754
0.470657
0.432699
0.538265
0.590190
0.435024
0.600083
0.529550
0.554898
0.690141
0.416274
0.743972
0.508990
0.350136
0.459760
0.444172
0.401909
0.518346
0.439181
0.721762
0.476372
0.452653
0.832692
0.614934
0.586838
0.199603
0.561444
0.621960
0.690203
0.572072
0.584385
0.470014
0.480894
0.495132
0.470927
0.470804
0.689840
0.702263
0.616715
0.273410
0.611090
0.575982
